### ISSUE TITLE
Campaign: Stop unpaid internships

### ISSUE THREAD

**Issue by Lingtax** [FreeOurKnowledge/documentation#12]
Internships are an occasional phenomenon in academia, notionally providing students the opportunity to get "valuable workplace / research experience" ahead of application for jobs or post-graduate training while also providing researchers an affordable (read: free) labour force. 

However, these practices are unethical, [entrenching privilege, suppressing diversity and inclusion, and devaluing labour](https://www.theatlantic.com/business/archive/2012/05/unpaid-internships-bad-for-students-bad-for-workers-bad-for-society/256958/), and in many jurisdictions, [these practices are also illegal](https://www.fairwork.gov.au/pay/unpaid-work/work-experience-and-internships) - skating by on lack of scrutiny and the perverse incentives to preserve the status quo (i.e. prospective interns are motivated to perpetuate the internship programs in order to get experience to remain competitive with students with internships).

We need to break this cycle, and it only happens through collective disarmament. In each field we need to agree to: 

1. Stop offering unpaid internship opportunities, despite the fact that it might hurt us
2. Stop considering unpaid internships in recruitment and selection.

**Comment by CooperSmout** [comment]
Thanks for posting @Lingtax. Agree with the sentiment, students are treated as free/cheap labour in academia and it's only possible because we collectively allow it. Same could be said for the high (and growing) ratio of phd positions to permanent positions, but I'll leave that for a separate thread/campaign :)

Three things stand out on first impression:

1. What sort of threshold are you thinking? If a big threshold (e.g. 1000s of pledges, or say 20% of a field), I think the campaign would have much more chance of success at a later stage of the project, when we've developed more of a following. The current OA campaigns haven't had much success, and I think a big reason for this is they set very large goals that seem unachievable. So I think the next few campaigns we post should have small targets, in the region of 10s or 100s of pledges, so we can reach threshold and demonstrate the concept in action. But here, if we were to use a small threshold (e.g. 100 pledges), I wonder if this would be big enough to have any real effect and also to protect those people who pledge.

2. Related to the above, would the campaign be locally specific (e.g. Australia only) or global? A local campaign could probably get away with smaller numbers, but unless I'm reading the [aust govt website](https://www.fairwork.gov.au/pay/unpaid-work/work-experience-and-internships) incorrectly it is currently **legal** to offer unpaid internships to students. 

3. How would we check whether people are compliant with their pledge? I think each action should be accountable in some way, so we can (1) demonstrate the effect we're having, (2) discourage people from breaking their pledge, and (3) reassure pledgers that they won't be acting alone when their pledge activate. But I can't think of any easy way to monitor either of the two actions you've proposed, particularly the latter ("Stop considering unpaid internships in recruitment and selection") which could operate implicitly even if people had pledged to ignore internships when hiring.

**Comment by Lingtax** [comment]
I think the PhD place issue is quite different, the solution there needs to be acknowledgement that a PhD is not a step to the academy, that is an overwhelming minority outcome and so that should be the lowest priority in training. 

Regarding your questions:

1. I think it needs to have a reasonable number behind it, and I think in practice there's more supportive sentiment than we might otherwise suspect. We just need to draw it out. Maybe mid-100s of signatories?
2. It's not intrinsically a regional problem, but the laws are. If it's international, we need to adjust the target into the 1000s though. Yes internships in coursework are ok, and fundamentally it's a different equation there (interns are guaranteed getting something [credit] from it). I think that then becomes a question of whether the unit was good or not, in which case there are sensible internal audit processes. It's the other internships that aren't associated with coursework AND are unpaid that are the potential problem. 
3. I think this is hard, but I'll keep thinking. Agree that there's still the risk of defection at play, but I'll keep thinking.

**Comment by CooperSmout** [comment]
> 1. I think in practice there's more supportive sentiment than we might otherwise suspect

I'll have to trust you on this one, since I'm not in a position to be hiring/offering internships and don't know many who are :)  I presume this would need to be a requirement for signing then -- some evidence that you can hire interns/ influence hiring decisions? And I also assume you're thinking to target researchers in psychology? 

> 2\. It's not intrinsically a regional problem, but the laws are

Agreed. I think this is the main obstacle to this campaign, because applicants can come from anywhere in the world. But perhaps this will be less of a concern in coming years. And at the very least, creating a local union to oppose these behaviours would send a strong signal. 

It would be good to start collating a list of eligibility requirements for signatories, to think precisely about who this campaign would target (by the way, most of these questions I'm asking are found in the [Readme](https://github.com/FreeOurKnowledge/documentation/blob/master/README.md), in case it helps organise thinking)

**Comment by AnthMHarris** [comment]
> I presume this would need to be a requirement for signing then -- some evidence that you can hire interns/ influence hiring decisions?

People of all career stages have the opportunity to work with unpaid interns. For example, a PhD student may be allocated an intern to help them collect data. My feeling is that decent pressure could be applied on lab-heads by PhD students, postdocs, etc. declining to work with interns if they are not being paid. So I dont think we need evidence that someone is a lab-head for their pledge to be worthwhile. 

Regarding the issue of legality, I think this can be left aside. Unpaid interns are an ethical issue, and an issue for the diversity of entrants into the field. Whether they are legal in a given area is beside the point. I think the pledge is worthwhile regardless.

Holding people accountable is hard, but making the names publicly available should go some way to this. With the names made public, any unpaid intern can see whether the person they are working for is claiming not to use unpaid interns. Accountability for hiring decisions is impossible for a variety of reasons, but I still think it is worth including that statement in a pledge with other aspects (for which people can be held accountable), if only to raise it in people's minds and draw attention to the whole problem. 

(apologies if any of this is rambling or incoherent. There is drilling going on upstairs and it is making it very hard to think.)

**Comment by CooperSmout** [comment]
> Holding people accountable is hard, but making the names publicly available should go some way to this. With the names made public, any unpaid intern can see whether the person they are working for is claiming not to use unpaid interns.

Fair point, and on reflection perhaps I'm being too strict in expecting actions to be public/accountable. My thinking on this was based on feedback about the OA campaigns (#5 #6 and #7), because people might hesitate to follow through with a high-risk boycott if its unclear whether other people will also follow through. But these new campaigns are a different beast, and I'm happy to ease back on that front and see how things evolve. At the end of the day, if we get pledges then that indicates people are happy enough with the campaign parameters, and there's plenty of other [reasons for pledging](https://github.com/FreeOurKnowledge/discussion/blob/master/marketing/reasons_to_pledge) that don't require actions to be trackable.
